DATA/UNIVERSE/SYSTEMS/Bw08/Bw08.ini

```diff
@@ -2242,3 +2242,19 @@ nickname = Bw08_ast_solar_beryllium_60_2
 pos = 9999, -40, 22485
 rotate = 90, 90, 0
 archetype = ast_solar_beryllium_60
+
+[Object]
+nickname = Bw06_suprise_Syd
+ids_name = 459951
+;res str
+; Salavager
+ids_info = 461130
+;res html
+; These are the remains of a junkers salvage ship. According to the records, the ship belonged to the pilot Thomas Fin.Due to a technical defect, he lost his way on the way to Bretonia until his life support systems finally failed.\n

```

Comment:
Junkers with capital J. Missing space after the `.`
"Lost his way on the way to Bretonia" sounds weird.